### PR TITLE
[update] Adding conda-forge installation instructions to README

### PR DESCRIPTION
**Pull request by ayushanand18**
## Overview
This PR aims to add `conda-forge` installation instructions to the `README`.

Addendum to #71.

Thanks!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -38,12 +38,37 @@ Installation
 ============
 
 Install from pypi
```
Comment (ocefpaf) [inline README.rst:40]:
```suggestion
Install from PyPI
```

File: README.rst [hunk 2]
```diff
@@ -38,12 +38,37 @@ Installation
...
+
+.. code-block:: console
+
+    conda install pyobis
+
+or with mamba:
+
+.. code-block:: console
+
+    mamba install pyobis
```
Comment (ocefpaf) [inline README.rst:66]: While this is 100% correct we don't want to be the source of truth for complex install configuration and options. You do have the link for further info right below this command.

```suggestion
.. code-block:: console

    conda install pyobis --channel conda-forge
```

File: README.rst [hunk 3]
```diff
@@ -38,12 +38,37 @@ Installation
...
+
+`More information here <https://github.com/conda-forge/pyobis-feedstock>`__
+
 Install latest dev version from github
```
Comment (ocefpaf) [inline README.rst:70]:
```suggestion
Install latest development version from GitHub
```

**Comment by ayushanand18** [comment]
Thanks a ton @ocefpaf for the review! I have updated those changes.

**Pull request merged**